src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapter.cs

```diff
@@ -269,19 +270,36 @@ public BluetoothLePacketType PacketType
         /// <exception cref="NotSupportedException">Thrown when the Bluetooth LE is not supported.</exception>
         /// <exception cref="InvalidOperationException">Thrown when the Bluetooth LE is not enabled.</exception>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated since API level 6. Please use GetServiceUuid() method on BluetoothLeDevice.")]
         public IEnumerable<string> ServiceUuid
         {
             get
             {
-                if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
-                {
-                    Log.Info(Globals.LogTag, "Retrieving Service uuid- ");
-                    return BluetoothLeImplAdapter.Instance.GetLeScanResultServiceUuids(_scanData, _packetType);
-                }
-                return null;
+                return GetServiceUuid(BluetoothLePacketType.BluetoothLeAdvertisingPacket);
```

Comment:
To provide a behavior consistency, you need to use _packetType
```suggestion
                return GetServiceUuid(_packetType);
```